docs/storybook/src/reference-assets/ChatMessage.mdx

```diff
@@ -0,0 +1,9 @@
+import * as ChatMessageStories from "./ChatMessage.stories";
+
+# ChatMessage Asset
+
+The `ChatMessage` asset is used as an example of async node.
```

Comment:
semantics, the `ChatMessage` itself isn't an async node, but it's an example of an asset that supports further async loading of future assets